great_expectations/data_context/types/base.py

```diff
@@ -1082,6 +1082,31 @@ def make_notebooks_config(self, data, **kwargs):
         return NotebooksConfig(**data)
 
 
+class ProgressBarsConfig(DictDot):
+    def __init__(
+        self,
+        globally: bool = True,
+        profilers: bool = True,
+        metric_calculations: bool = True,
+    ):
+        self.globally = globally
+        self.profilers = profilers
+        self.metric_calculations = metric_calculations
+
+    def is_enabled(self, key: str) -> bool:
+        if super().__getitem__("globally") is False:
+            return False
+        elif super().__getitem__(key) is False:
+            return False
+        return True
```

Comment:
Am I getting this relationship correct? Global has the ability to shut down everything even if profilers or metric_calculations is enabled?